src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -90,9 +95,12 @@
         timestamp-str]])))
 
 (defview quoted-message
-  [_ reply]
-  [react/view {:style (style/quoted-message-container)}
-   [components.reply/reply-message reply false]])
+  [_ reply pin?]
+  [react/view {:style (when-not pin? (style/quoted-message-container))}
+   [components.reply/reply-message reply false pin?]])
+
+(defn is-system-text? [content-type]
```

Comment:
I think predicates in Clojure never use the `is-`, `are-`, etc prefixes because the question mark is a strong convention, especially for function predicates.